src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content-datasource/main.js

```diff
@@ -78,7 +78,7 @@ define([], function() {
                     selected: this.selected,
                     actionCallback: function(item) {
                         this.selected = item.id;
-                        this.options.selectCallback(item.id, item.path, item.title);
+                        this.options.selectCallback(item.id, item.path, item.title, item);
```

Comment:
Is it really necessary to pass the entire item as the last parameter? If it is, would it be a too big BC break to remove the other three arguments?